docs/getting-started/index.md

```diff
@@ -2,9 +2,15 @@
 title: "Getting Started"
 ---
 
-This is the high level documentation for Substrate, a next generation blockchain development framework.
+Welcome to the Substrate Developer Hub. 
 
-You might be looking for the reference documentation for Substrate's Rust codebase. You can find those `rustdocs` in our [Reference Documentation](/rustdocs/).
+As a next generation blockchain framework, Substrate is designed to help majority of developers build decentralized application to improve their business value. 
+
+There are two ways to build decentralized application in Substrate ecosystem, one is to build your own business specific blockchain, the other is to run your business with Smart Contract. As you walk through the documents, you will learn more details.
```

Comment:
What about Substrate based front end experiences?

I would be hesitant to add any lines like "There are two ways to use Substrate"...

Reply:
* I believe front end experiences can be merged into the generic blockchain development and smart contract development.
* Already refactor `There are two ways to use Substrate...`